cricket/view.py

```diff
@@ -162,6 +162,7 @@ def _setup_menubar(self):
         # self.menu_file.add_command(label='New', command=self.cmd_dummy, accelerator="Command-N")
         # self.menu_file.add_command(label='Open...', command=self.cmd_dummy)
         # self.menu_file.add_command(label='Close', command=self.cmd_dummy)
+        self.menu_file.add_command(label='Quit', command=self.cmd_quit) if sys.platform !='darwin' else None
```

Comment:
I think the `if COND then VAL1 else VAL2` makes sense when you're using the result of the if as a value.

In this case, it will be more readable to have the classic if.

@nfredrik can you update this? I'm missing the feature as well. :)